wcvendors/wcvendors
# Add feature to assign coupons to vendor.

### All Submissions:


### Changes proposed in this Pull Request:

Add a new feature to assign coupons to vendors. Admin can choose a vendor in a single coupon edit screen or use bulk edit to assign a vendor to multiple orders at once.

Closes wcvendors/wcvendors-pro#1692 

### How to test the changes in this Pull Request:

Single coupon edit screen.
1. Add or create a coupon as an administrator
2. Check that there is a 'Vendor' meta box with a vendor select field
3. Search, select vendor, and publish or update coupon.
4. Check the assigned vendor was saved and preselected after saving.

Bulk coupon edit screen.
1. Go to the Coupons page, and select multiple coupons by checking the corresponding checkboxes or the select all checkbox.
2. Select 'Edit' in bulk actions and click 'Apply'
3. Check that there is a 'Vendor' dropdown
4. Search, select vendor, and click update to perform the bulk update.
5. Check that all the selected coupons have been assigned to the selected vendor.

Check the UI is consistent for the Product and Coupon edit screen
1. Check single coupon screen shows the same Vendor meta box as the single coupon screen
2. Use bulk edit coupons and bulk edit products to check the Vendor select box looks and works the same

### Other information:
